openerp/tools/translate.py

```diff
@@ -776,17 +842,15 @@ def get_root_view(xml_id):
                 except (IOError, etree.XMLSyntaxError):
                     _logger.exception("couldn't export translation for report %s %s %s", name, report_type, fname)
 
-        for field_name, field_def in obj._columns.items():
-            if model == 'ir.model' and field_name == 'name' and obj.name == obj.model:
-                # ignore model name if it is the technical one, nothing to translate
-                continue
-            if field_def.translate:
+        for field_name, field_def in obj._fields.iteritems():
+            if getattr(field_def, 'translate', None):
                 name = model + "," + field_name
                 try:
-                    term = obj[field_name] or ''
+                    value = obj[field_name] or ''
                 except:
```

Comment:
the `try/except` dates from 2009; apparently, the field access was sometimes failing...